transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformerProvider/build.gradle

```diff
@@ -16,7 +16,6 @@ dependencies {
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core'
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
     implementation group: 'org.slf4j', name: 'slf4j-api'
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl'
```

Comment:
Awesome - thanks for maintaining the low numbers of dependencies in our libraries